packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -336,10 +336,12 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderProps, IDetai
   private _onDrop(item?: any, event?: DragEvent): void {

```

Comment:
Stop making everything `optional`. Prefer everything to be required unless the caller truly can't supply it.